File: Servidor/AtHomeTFGCarlosHernandez/src/main/java/org/tfg/athometfgcarloshernandez/spring/model/response/GetUsuariosResponseDTO.java
package org.tfg.athometfgcarloshernandez.spring.model.response;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.tfg.athometfgcarloshernandez.spring.model.UsuarioDTO;

import java.util.List;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class GetUsuariosResponseDTO {
    private List<UsuarioDTO> usuarios;
}

File: Cliente/AtHomeTFGAndroidCarlosHernandez/app/src/main/java/com/example/athometfgandroidcarloshernandez/data/model/response/GetUsuariosResponseDTO.kt
package com.example.athometfgandroidcarloshernandez.data.model.response

import com.example.athometfgandroidcarloshernandez.ui.framework.screens.inmuebles.InmueblesContract

data class GetUsuariosResponseDTO (
    val usuarios: List<InmueblesContract.UsuarioInmuebles>
)